lib/src/base-styles/_nx-btn.scss

```diff
@@ -101,6 +101,34 @@
       height: 16px;
     }
   }
+
+  &--small {
+    font-size: var(--nx-font-size-s);
+    padding: var(--nx-spacing-base) var(--nx-spacing-3x);
+    margin: 0 var(--nx-spacing-3x) 0 0;
+    border-color: var(--nx-color-small-button-border);
+    color: var(--nx-color-small-button-text);
+
+    .nx-icon {
+      height: var(--nx-spacing-3x);
+      margin-right: var(--nx-spacing-2x);
```

Comment:
Should be redundant with `.nx-icon`'s default margin

Reply:
https://github.com/sonatype/sonatype-react-shared-components/pull/1220/commits/1a8d7d3e82a75132d6e193e3bae6877b22181660